modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -2365,19 +2373,36 @@ private void onCacheStarted(GridCacheContext cacheCtx) throws IgniteCheckedExcep
             desc.schema() != null ? desc.schema() : new QuerySchema(), desc.sql());
 
         if (log.isInfoEnabled()) {
+            String expPlcInfo = buildExpirePolicyInfo(cacheCtx);
+
             log.info("Started cache in recovery mode [name=" + cfg.getName() +
                 ", id=" + cacheCtx.cacheId() +
                 (cfg.getGroupName() != null ? ", group=" + cfg.getGroupName() : "") +
                 ", dataRegionName=" + dataRegion +
                 ", mode=" + cfg.getCacheMode() +
                 ", atomicity=" + cfg.getAtomicityMode() +
                 ", backups=" + cfg.getBackups() +
-                ", mvcc=" + cacheCtx.mvccEnabled() + ']');
+                ", mvcc=" + cacheCtx.mvccEnabled() +
+                (expPlcInfo != null ? ", " + expPlcInfo : ", expiryPolicy=null") + ']');
         }
 
         return cacheCtx;
     }
 
+    /**
+     * Build formatted string with expire policy info.
+     *
+     * @param cacheCtx - cache context.
+     * @return formatted expire policy info.
+     */
+    private String buildExpirePolicyInfo(@NotNull GridCacheContext cacheCtx) {
```

Comment:
Right now this method returns either string with full information about expiry policy or null, this forces us to use cumbersome `?:` operator to substitute `expiryPolicy=null` message.

Lets change logic of this method in such a way that it will return both messages: full info and null info depending on settings.

Reply:
Please rewrite the method so that it always returns a string: `expiryPolicy=null` or `expiryPolicy=[type=Klass, isEagerTtl=true]`. Don't use `String.format`